service/start.go

```diff
@@ -0,0 +1,38 @@
+package service
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+	"tat_gogogo/utilities/logs"
+
+	"github.com/spf13/viper"
+)
+
+func Start() {
```

Comment:
public function should at least have comment
the format will be `[Function name] is ........`, so in this case it will be `Start is ......`

Reply:
Done